maxscale/values.yaml

```
@@ -28,6 +28,12 @@ fullnameOverride: ""
 service:
   type: ClusterIP
   port: 3306
+  masteronly:
+    type: LoadBalancer
+    annotations: {}
+  rwsplit:
+    type: LoadBalancer
+    annotations: {}
```

Please keep the original value, otherwise this would be a breaking change
```suggestion
  masteronly:
    type: ClusterIP
    annotations: {}
  rwsplit:
    type: ClusterIP
    annotations: {}
```

---

Resolved